Documentation/Signoffs/Braking-Subsystem-Dylan-Scruggs.md

```diff
@@ -0,0 +1,68 @@
+
+### Braking Resistor Analysis
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Due to this system operating on regenerative braking, current backflow will be present into the VFD. Therefore, operation must be put into place in order to dissipate the generated power. This will be done by connected a braking resistor to the DC Bus terminals of the VFD in order to dissipate this generated energy through heat. This is required when using a VFD, with a motor acting as a generator, due to the power back feed. This reverse flow into the VFD, if not properly dissipated, will cause the drive’s DC bus voltage to rise, resulting in over-voltage faults. This will lead to the drive protecting itself by shutting down, or possibly lead to damage to the input rectifiers. <br><br>
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;In order to select and size the chosen braking resistor, 2 main factors were taken into account. The first parameter is the minimum resistance value. This value won’t affect the resistor’s ability to dissipate the incoming power; however, it is vital to making sure it is compatible with the chosen VFD. This compatibility lies within the VFD’s internal brake chopper circuitry. This circuitry consists of a brake transistor that shunts excess current from the DC bus to the braking resistor. However, this transistor exhibits a maximum current rating listed within the datasheet. With this current rating and the over-voltage level of the VFD, the minimum rating of the resistor can be found through ohm’s law. The over-voltage level for the selected resistor is 820VDC. This value is 130%-150%  of the drive’s nominal dc bus voltage threshold. Therefore, the nominal dc bus voltage level is between 550 VDC and 630VDC. This puts the minimum resistance value of the resistor at 34.6Ω. The second parameter is the power dissipation capacity. This rating will ensure that the resistor is capable of handling the power sent to it from the VFD. For this rating, the maximum power of the bike is used (3500 W). With these required parameters, the Ironhorse GS-BR-1K5W043 braking resistor was selected. This resistor’s ratings for resistance and peak power are, respectively, 49 Ω and 12.4 kW. This component not only meets the required ratings, but also is the recommended component to use with the previously selected VFD. With these ratings, this component also meets the requirements of constraint 24.<br><br>
+
+### E-Stop Analysis
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Customer request and standard NFPA 79 – 10.7.1. require that an emergency stop be present to stop in the event of dangerous situations. This requirement will be met through the coaction of this system lowering the frequency to the motor to its minimum value and a mechanical brake designed and integrating by the mechanical team. With this implementation, the system will have the ability to bring components to a quick stop. Furthermore, this capability also meets the requirements of constraint 5.<br><br>
```

Comment:
You need to add an estop on the run input of the drive for quick shutdown.

Reply:
I have addressed this issue in the newly submitted signoff request.